packages/mobile/src/utils/formatting.ts

```diff
@@ -10,7 +10,8 @@ export const getMoneyDisplayValue = (
   includeSymbol: boolean = false,
   roundingTolerance: number = 1
 ): string => {
-  const decimals = CURRENCIES[currency].displayDecimals
+  // TODO: remove the hardcoded ternary operation here when displayDecimals is updated from monorepo
```

Comment:
The `CURRENCIES` enum is being moved to the wallet repo 😁 https://github.com/celo-org/wallet/pull/318/files#diff-2687e3b263090b669733a77c6d0020649f4f39eca3b8a00bfbe164f5c44d7a95